supplements/tashas-cauldron-of-everything/race-custom-lineage.xml

```diff
@@ -26,7 +26,8 @@
 			<select type="Ability Score Improvement" name="Ability Score Improvement +2 (Custom Lineage)" supports="Custom Ability Score Increase 2" />
 			<select type="Feat" name="Feat (Custom Lineage)" />
 			<select type="Racial Trait" name="Variable Trait" supports="Custom Lineage Variable Trait" />
-			<grant type="Language" id="ID_LANGUAGE_COMMON" />
+			<grant type="Language" id="ID_LANGUAGE_COMMON" requirements="!ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
+			<select type="Language" name="Customized Language" supports="Custom Race Language" requirements="ID_WOTC_TCOE_OPTION_CUSTOMIZED_LANGUAGE" />
```

Comment:
i don't believe rules for Custom Languages extend to Custom Lineage though, Custom Rules are applied to races that you already exist, i believe, while Custom Lineage is just option to replace your race with. So i'm not really sure if we should use it... i mean, there probably are people who would like ability to replace Common to something else, but i'm not sure if that's the intention of the book.